### ISSUE TITLE
Review best versification for the SWORD module build

### ISSUE THREAD

**Issue by DavidHaslam** [krisek/HunKar#1]
1. During build using **osis2mod** with `-v KJV ` in the command line, there are **105** V11N warnings / append notifications, viz.

[osis2mod.log](https://github.com/krisek/HunKar/files/5057783/osis2mod.log)

2. For the module thus made, the **emptyvss** utility outputs **36** verse references with no text, viz.

[HunKar.emptyvss.txt](https://github.com/krisek/HunKar/files/5057780/HunKar.emptyvss.txt)

In the light of both observations, please review whether any of the [Alternate Versifications](https://wiki.crosswire.org/Alternate_Versification) would be more suited for the module.

Also review the **osisID** tags in the XML file against the verse numbers in a reference printed copy of the **Karoli Bible**, just in case some of these need to be corrected.

**Comment by krisek** [comment]
I checked this now in detail, any versification I try, I get only worse results. What do you think what could I do to resolve this?

**Comment by DavidHaslam** [comment]
That might mean one of two things.
1. The printed edition and the present electronic edition have some hitherto undetected differences in versification.
2. The versification for the Karoli Bible is different from all those so far supported by SWORD.

The first can only be ruled out by diligent checking.

**Comment by DavidHaslam** [comment]
It may be of interest to know that the earlier version of the **HunKar** module that I analysed in **2011** included **parenthesised inline references** for the verses that were **appended** because they were outside the **KJV** versification.

[Hunkar.diatheke.append.txt](https://github.com/krisek/HunKar/files/5064117/Hunkar.diatheke.append.txt)

A few of these verses have _multiple_ appends. The total was 105 - as noted earlier.

**Comment by krisek** [comment]
Finally I set German versification in 99ff41d, I think this fits the best. (ie. it renders the same as HunUj)

**Comment by DavidHaslam** [comment]
@krisek 

Please then add the following to **hunkar.conf**
```
Versification=German
```
providing this change works OK. I'm about to test it.

**Comment by DavidHaslam** [comment]
**German** versification causes more mismatches than with **KJV**.

The attached Zip file contains the files `osis2mod.log` & `HunKar.emptyvss.text` after building with `-v German ` in the CL.

[Analysis3.zip](https://github.com/krisek/HunKar/files/5119812/Analysis3.zip)

Please revert and re-open this issue. @krisek

**Comment by krisek** [comment]
It is very interesting, since as soon as I switched to German versification Xiphos and Bibletime started to render it properly. So even though there are more mismatches, it still works better, very weird.

![versification_compare](https://user-images.githubusercontent.com/786044/91093355-997ff880-e659-11ea-8eed-fe155197839e.png)

**Comment by DavidHaslam** [comment]
@krisek 

On a hunch, knowing that the **Hungarian Reformed Church** had historic connections with Geneva and Calvin, I tried a module build using `-v Calvin` in the **osis2mod** command line.

The good news is that there are significantly _fewer_ append warnings in `osis2mod.log` compared to using `-v KJV`.

- KJV has 105 appends
- Calvin has only 42 appends

The significant improvement is mainly due to the better matching for how the Psalms with canonical titles are versified.

The attached Zip file contains the files `osis2mod.log`  for `-v Calvin` & `HunKar.emptyvss.txt` for `Versification=Calvin`.

[Analysis5.zip](https://github.com/krisek/HunKar/files/5124215/Analysis5.zip)

The attached Zip file contains the files `osis2mod.log`  for `-v KJV` & `HunKar.emptyvss.txt` for `Versification=KJV`.

[Analysis4.zip](https://github.com/krisek/HunKar/files/5124249/Analysis4.zip)

cf. The **emptyvss** statistics are also compared:
- KJV has 36 empty verses
- Calvin has 41 empty verses

**Notes:**
1. `Calvin` was one of the 3 Alternate Versification schemes for **French Bibles** added in SWORD v1.8
2. Currently, **PocketSword** v1.4.7 was compiled using SWORD v.1.7.x - so any Bible module with `Versification=Calvin` will not be properly supported by PocketSword.
3. The other 2 **French Bible** Versification schemes added in SWORD v1.8 (`DarbyFr` & `Segond`) are later and much less likely to be applicable.

**Comment by DavidHaslam** [comment]
NB. The test builds were made without changing the following lines in the XML file.
```
                <work osisWork="HunKar">
                        <title>Magyar Revideált Károli</title>
                        <identifier type="OSIS">Bible.HunKar</identifier>
                        <refSystem>Bible.German</refSystem>
                </work>
                <work osisWork="defaultReferenceScheme">
                        <refSystem>Bible.German</refSystem>
                </work>
```

There are clearly problems in how Xiphos fails to display the **section headings** correctly with both **Calvin** and **KJV**, and I am now wondering if this may be due to having not changed the `<refSystem>...</refSystem>` entries.

_I think we need advice from the SWORD developers on this_.

**Comment by krisek** [comment]
Very good catch, many thanks, actually I [checked](https://github.com/krisek/HunKar/issues/1#issuecomment-672148470) Calvin versification and it didn't work --- maybe I messed up something on my build system

(HunUj can be reconsidered too)

fixed in 8471ba0

**Comment by DavidHaslam** [comment]
@krisek 

This line in `hunkar.conf` ought to mention that the **Versification** has been changed from the default KJV.
```
History_2.0=Module rebuilt based on new source with cross-references (from http://szentiras.hu/KG) and section titles (from http://abibliamindenkie.hu/karoli). Description corrected.
```
